.github/ISSUE_TEMPLATE/1_Bug_report.yaml

```
@@ -0,0 +1,41 @@
+      attributes:
+          label: Description
+          description: A clear and concise description of the problem
+      validations:
+          required: true
+    - type: textarea
+      id: how-to-reproduce
+      attributes:
+          label: How to reproduce
+          description: |
+              ⚠️  This is the most important part of the report ⚠️
+              We’ll have a much better chance of helping if we can easily reproduce the issue. 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)